README.md

```diff
@@ -1,39 +1,90 @@
 # Dice
+## Table of Contents
 
-"Кубики"
+- [Dice] (# dice)
```

Comment:
![image](https://user-images.githubusercontent.com/22330612/91823256-99778e00-ec41-11ea-9792-539f22c84a54.png)

![image](https://user-images.githubusercontent.com/22330612/91823304-abf1c780-ec41-11ea-8109-3b5648885816.png)

it seems, markdown format was broken while translation